examples/web/index.html

```diff
@@ -308,58 +308,42 @@ <h3 class="typography-heading-small-text">Stacked blocks</h3>
         <div class="Container">
           <h2 class="typography-heading-medium-text">Helpers</h2>
           <h3 class="typography-heading-small-text">Typography</h3>
-          <p class="typography-heading-xlarge-text">Heading XLarge Text</p>
-          <p><a href="#" class="typography-heading-xlarge-link">Heading XLarge Link</a></p>
+          <p class="typography-body-large-button">Body Large Button</p>
+          <p><a href="#" class="typography-body-large-link-no-underline">Body Large Link No Underline</a></p>
+          <p><a href="#" class="typography-body-large-link-underline">Body Large Link Underline</a></p>
+          <p class="typography-body-large-text-bold">Body Large Text Bold</p>
+          <p class="typography-body-large-text-italic">Body Large Text Itaic</p>
```

Comment:
```suggestion
          <p class="typography-body-large-text-italic">Body Large Text Italic</p>
```

More occurrences below.

Reply:
😨 how did this happen, thanks for noticing